pkg/ccl/crosscluster/logical/lww_row_processor.go

```diff
@@ -614,17 +615,20 @@ func insertColumnNamesForFamily(
 			return nil
 		}
 		colName := col.GetName()
-		// We will set crdb_replication_origin_timestamp ourselves from the MVCC timestamp of the incoming datum.
-		// We should never see this on the rangefeed as a non-null value as that would imply we've looped data around.
-		if colName == originTimestampColumnName {
+		// We will set crdb_replication_origin_timestamp ourselves from
+		// the MVCC timestamp of the incoming datum. We should never see
+		// this on the rangefeed as a non-null value as that would imply
+		// we've looped data around.
+		//
+		// TODO(ssd): If we aren't going to support a migration path, we
```

Comment:
yeah, let's remove this completely. i see no need for a migration path. We could do this in a follow up though.

Reply:
Removed this in a new commit.